stdlib/postgres/postgres.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package postgres
```

Comment:
stdlib doesn't make sense as a package name to me. It seems like it should be "dbsql" or "hooks".

Reply:
How about `sqldriver`?

Reply:
Matches with this page: https://github.com/golang/go/wiki/SQLDrivers.

Reply:
Or better yet: `driver` since it's a SQL library.

Reply:
But that would conflict with the standard library driver package, so never mind. `sqldriver` seems best.